From 9b5dfd06509ad450c6e9ece747d67346a5b5e6a9 Mon Sep 17 00:00:00 2001
From: "shravanrn@gmail.com" <shravanrn@gmail.com>
Date: Mon, 11 Nov 2019 09:19:29 +0000
Subject: [PATCH 07/29] Bug 1576054 - Use forked rust/nix that removes
 references to newer libc functions setns, process_vm_writv, process_vm_readv.
 References to these functions create missing symbols in gkrust, which is
 configured to a build failure in Firefox r=froydnj

Differential Revision: https://phabricator.services.mozilla.com/D44341

--HG--
extra : moz-landing-system : lando

diff --git a/Cargo.toml b/Cargo.toml
index 7a0ba6d7abad..ef0649707d95 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -61,6 +61,7 @@ libudev-sys = { path = "dom/webauthn/libudev-sys" }
 serde_derive = { git = "https://github.com/servo/serde", branch = "deserialize_from_enums10" }
 winapi = { git = "https://github.com/froydnj/winapi-rs", branch = "aarch64" }
 packed_simd = { git = "https://github.com/hsivonen/packed_simd", branch = "rust_1_32" }
+nix = { git = "https://github.com/shravanrn/nix/", branch = "r0.13.1", rev="4af6c367603869a30fddb5ffb0aba2b9477ba92e" }
 
 [patch.crates-io.cranelift-codegen]
 git = "https://github.com/CraneStation/Cranelift"
-- 
2.25.1

